docs/dev/feature-usage.md

```diff
@@ -19,6 +19,16 @@ the customer added.
 
 ***SetUsage*** is a helper function. When calling it you should specify the following parameters: 
 
+> Important note:
```

Comment:
@slaviered lets add how to verify it in ELK?

Reply:
OK. When the links of ELK are final (they are on transition now).